src/comparisons.jl

```diff
@@ -1,7 +1,42 @@
-# This file is inspired from Polyhedra.jl
+# Some functions in this file are inspired from Polyhedra.jl
 
-const ABSZTOL(N::Type{<:AbstractFloat}) = N(10) * sqrt(eps(N))
-const ABSZTOL(N::Type{Rational{INNER}}) where {INNER} = zero(N)
+# struct to contain the tolerances for a given numberic type
+mutable struct TOL{N<:Number}
+    rtol::N
+    ztol::N
+    atol::N
+end
+
+# global Float64 tolerances
+const F64 = Float64
+const tF64 = Type{Float64}
+const _TOL_F64 = TOL(Base.rtoldefault(F64), F64(10)*sqrt(eps(F64)), zero(F64))
+
+get_rtol(N::tF64) = _TOL_F64.rtol
+get_ztol(N::tF64) = _TOL_F64.ztol
+get_atol(N::tF64) = _TOL_F64.atol
+
+set_rtol(N::tF64, ε::F64) = _TOL_F64.rtol = ε
+set_ztol(N::tF64, ε::F64) = _TOL_F64.ztol = ε
+set_atol(N::tF64, ε::F64) = _TOL_F64.atol = ε
+
+# global rational tolerances
+const RAT = Rational{Int}
```

Comment:
Let us use `Rational` instead so it automatically works for other `Rational` instantiations as well.

Reply:
Fixed; i used `Type{<:Rational}` where possible.

Reply:
Note that `typeof(zero(Rational)) == Rational{Int}`.